### ISSUE TITLE
Relicense to GPL2+?

### ISSUE THREAD

**Issue by Freso** [craigshelley/MusicBrainzPicardPlugin_FirstRecorded#5]
Any chance you can change the license for the plugin to be "GPL version 2 *or, at your discretion, any later version*"? Picard itself may need to switch to GPL 3(+) for the Picard 2.0 code (due to a license change in one of the dependencies), and thus plugins may also need to change to a compatible license for this.

**Comment by Freso** [comment]
@craigshelley Hi. :) This plugin has just come up in discussions on the MetaBrainz forums again, and I or someone else may look into trying to get it into the official picard-plugins repository. Any chance you could take a look at relicensing?

**Comment by craigshelley** [comment]
Yes, I don't mind re-licencing as GPL 2 / 3 / 2+. I would prefer the plugin
to be included in the official repository and used rather than fade into
obscurity due to a licencing issue.
Is there anything I need to do?
--
Craig

On 15 Jan 2019 13:26, "Frederik “Freso” S. Olesen" <notifications@github.com>
wrote:

@craigshelley <https://github.com/craigshelley> Hi. :) This plugin has just
come up in discussions on the MetaBrainz forums again, and I or someone
else may look into trying to get it into the official picard-plugins
repository. Any chance you could take a look at relicensing?

—
You are receiving this because you were mentioned.

Reply to this email directly, view it on GitHub
<https://github.com/craigshelley/MusicBrainzPicardPlugin_FirstRecorded/issues/5#issuecomment-454391233>,
or mute the thread
